doc/Requirements Specification/Parts/Non-Functional Requirements.lyx

```diff
@@ -858,7 +870,20 @@ To analyse the entire project, there is an entry
 \begin_inset Quotes eld
 \end_inset
 
-Analyse with Beagle
+Analyse with 
+\begin_inset ERT
+status open
+
+\begin_layout Plain Layout
+
+
+\backslash
+gls{Beagle}
+\end_layout
```

Comment:
A user can have multiple tools, which offer a way to analyse a project.
So I think it's best practice, to mention Beagle in the context menu entries.